src/core/dbus-execute.c

```diff
@@ -1229,6 +1256,7 @@ const sd_bus_vtable bus_exec_vtable[] = {
         SD_BUS_PROPERTY("LogRateLimitIntervalUSec", "t", bus_property_get_usec, offsetof(ExecContext, log_ratelimit_interval_usec), SD_BUS_VTABLE_PROPERTY_CONST),
         SD_BUS_PROPERTY("LogRateLimitBurst", "u", bus_property_get_unsigned, offsetof(ExecContext, log_ratelimit_burst), SD_BUS_VTABLE_PROPERTY_CONST),
         SD_BUS_PROPERTY("LogExtraFields", "aay", property_get_log_extra_fields, 0, SD_BUS_VTABLE_PROPERTY_CONST),
+        SD_BUS_PROPERTY("LogFilterPatterns", "as", property_get_log_filter_patterns, 0, SD_BUS_VTABLE_PROPERTY_CONST),
```

Comment:
Please expose properly structured data, i.e. the inversion boolean should be a boolean field in a structure, i.e. `a(bs)` or so would be appropriate I think.

Reply:
So we would read strings (with or without `~` prefix) from unit files, but expose a set of structs?